src/libs/actions/Policy.js

```diff
@@ -1000,6 +1001,38 @@ function openWorkspaceInvitePage(policyID, clientMemberEmails) {
     });
 }
 
+/**
+ *
+ * @param {String} reportID
+ */
+function leaveRoom(reportID) {
+    API.write('LeaveRoom', {
+        reportID,
+    }, {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.SET,
+                key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+                value: {
+                    stateNum: CONST.REPORT.STATE_NUM.SUBMITTED,
+                    statusNum: CONST.REPORT.STATUS.CLOSED,
+                },
+            },
+        ],
+        failureData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.SET,
+                key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+                value: {
+                    stateNum: CONST.REPORT.STATE_NUM.OPEN,
+                    statusNum: CONST.REPORT.STATUS.OPEN,
+                },
+            },
+        ],
+    });
+    Report.navigateToConciergeChat();
```

Comment:
In the end, I think it's better to navigate to Concierge (like we do [here](https://github.com/Expensify/App/blob/b703eac7951ba946ba63e3fc83e6fe1b5c64f77d/src/pages/home/report/ReportActionItemCreated.js#L52)) instead of just showing the `Hmm... it's not here` message.